tensorboard/plugins/profile/pod_viewer/stack_bar_chart/stack-bar-chart.ts

```diff
@@ -98,8 +117,12 @@ Polymer({
         .selectAll('rect').data((d) => d)
         .enter().append('rect')
         .attr('width', xScale.bandwidth())
-        .attr('y', (d) => yScale(d[1]))
-        .attr('height', (d) => yScale(d[0]) - yScale(d[1]))
+        .attr('y', (d) => yScale(d[1]) ? yScale(d[1]) : 0)
+        .attr('height',
+            (d) => {
+               const h = yScale(d[0]) - yScale(d[1]);
+               return h ? h : 0;
```

Comment:
This is for backward compatibility, because I have removed crsDurationUs, and added allReduceComputeDurationUs and allReduceSyncDurationUs, older traces couldn't find this two layers, and h is not defined.

Reply:
Might want to add a line for unwary future readers.